src/frontend/scss/generic/_accessibility.scss

```diff
@@ -1,6 +1,44 @@
 // Helper class to hide elements from sighted users and have accessibility devices still read them
 // to users that need it.
-.offscreen {
+
+// Screen reader only content
+.offscreen,
+.sr-only {
+  border: 0;
+  clip: rect(1px, 1px, 1px, 1px);
+  clip-path: inset(50%);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
   position: absolute;
-  left: -9999rem;
+  width: 1px;
+  word-wrap: normal !important;
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border: 0;
+
+  // Screen reader toggleable function, including offscreen navigation
+  &:active,
+  &:focus {
+    clip: auto !important;
+    clip-path: none;
+    display: block;
+    font-size: 1em;
+    height: auto;
+    left: 5px;
+    line-height: normal;
+    text-decoration: none;
+    top: 5px;
+    width: auto;
+    z-index: 100000; /* Above WP toolbar. */
+    background-color: r-color('midnightblue');
```

Comment:
I think a lot of these styles should not be on this class. This should just "cancel out" `sr-only` IMO and put the element back in the flow of the page.

Then we can make a class for `.skip-to-content` to add styles that are specific to this exact link

Reply:
Ok. Will do these changes. So we'll stay with the offscreen, remove `sr-only` and implemente `.skip-to-content` with button specific styling? Does this sound good? If so, I'll do that.

Reply:
We can keep the `sr-only` name, IMO it is clearer than `offscreen`, the only downside is that it needs a search-and-replace to remove existing `offscreen` uses.

Apart from that, sounds great!

Reply:
Exactly that @mbenadda. Will fix that ASAP. Thanks for all the help.